papers/numba/looplifting/looplifting.rst

```diff
@@ -0,0 +1,157 @@
+Loop Lifting
+------------
+
+In a real application, functions may depend on features that Numba has no
+optimization strategy.  Such functions will be compiled by Numba using the
```

Comment:
Is there a word missing? "that Numba has no optimization strategy for", perhaps?